jolt-core/src/benches/bench.rs

```diff
@@ -123,62 +115,63 @@ fn halo2_comparison_benchmarks() -> Vec<(tracing::Span, Box<dyn FnOnce()>)> {
     ]
 }
 
-#[rustfmt::skip]
 fn rv32i_lookup_benchmarks() -> Vec<(tracing::Span, Box<dyn FnOnce()>)> {
-  let mut rng = test_rng();
-
-  let mut ops: Vec<RV32I> = vec![
-    RV32I::ADD(ADDInstruction(rng.next_u32() as u64, rng.next_u32() as u64)),
-    RV32I::AND(ANDInstruction(rng.next_u32() as u64, rng.next_u32() as u64)),
-    RV32I::BEQ(BEQInstruction(rng.next_u32() as u64, rng.next_u32() as u64)),
-    RV32I::BGE(BGEInstruction(rng.next_u32() as u64, rng.next_u32() as u64)),
-    RV32I::BGEU(BGEUInstruction(
-      rng.next_u32() as u64,
-      rng.next_u32() as u64,
-    )),
-    RV32I::BLT(BLTInstruction(rng.next_u32() as u64, rng.next_u32() as u64)),
-    RV32I::BLTU(BLTUInstruction(
-      rng.next_u32() as u64,
-      rng.next_u32() as u64,
-    )),
-    RV32I::BNE(BNEInstruction(rng.next_u32() as u64, rng.next_u32() as u64)),
-    RV32I::JAL(JALInstruction(rng.next_u32() as u64, rng.next_u32() as u64)),
-    RV32I::JALR(JALRInstruction(
-      rng.next_u32() as u64,
-      rng.next_u32() as u64,
-    )),
-    RV32I::OR(ORInstruction(rng.next_u32() as u64, rng.next_u32() as u64)),
-    RV32I::SLL(SLLInstruction(rng.next_u32() as u64, rng.next_u32() as u64)),
-    RV32I::SRA(SRAInstruction(rng.next_u32() as u64, rng.next_u32() as u64)),
-    RV32I::SRL(SRLInstruction(rng.next_u32() as u64, rng.next_u32() as u64)),
-    RV32I::SUB(SUBInstruction(rng.next_u32() as u64, rng.next_u32() as u64)),
-    RV32I::XOR(XORInstruction(rng.next_u32() as u64, rng.next_u32() as u64)),
-  ];
-  for _ in 0..15 {
-    ops.extend(ops.clone());
-  }
-  println!("Running {:?}", ops.len());
+    let mut rng = rand::rngs::StdRng::seed_from_u64(1234567890);
+
+    const NUM_CYCLES: usize = 64_000;
```

Comment:
We should just make this a flag eventually. Very common workflow to run different sizes back to back (without rebuilding).